rfcs/0019-semantic-versioning.md

```diff
@@ -0,0 +1,132 @@
+- Jenkins build versions serve as a reference point to specific job executions, but are otherwise meaningless (`#260` vs `#261`)
+
+Though helpful, existing resource versions are insufficient for deriving relevant information for the change outside the specific contexts in which the versions are applied; in other words, these version strings are not _semantic_.
+However, it's possible to generate a more meaningful, top-level version string that can serve as a heuristic to facilitate managing, documenting, and communicating changes among BFD contributors and customers alike.
+
+## Proposed Solution
+[Proposed Solution]: #proposed-solution
+
+BFD uses a version string that conforms to [semver.org@2.0.0](https://semver.org/spec/v2.0.0.html#summary).
+This version string is applied to various release artifacts that can be used by contributors and customers to facilitate communications surrounding changes across the BFD system.
+
+### Proposed Solution: Detailed Design
+[Proposed Solution: Detailed Design]: #proposed-solution-detailed-design
+
+1. A BFD contributor as _release engineer_ will identify a specific point in the _repository's history as being important_ by creating a [Git Tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging).
+2. The git tag will conform to the [semver@2.0.0 specification](https://semver.org/spec/v2.0.0.html#semantic-versioning-specification-semver) including precedence rules for constructing the _next_ version from the _current_/_latest_ tagged release version.
+3. Specific to the BFD system, a release intended for general availability (GA) will match the latest version of the BFD FHIR API endpoint that is enabled by default;
+that is, the latest versioned BFD FHIR API endpoint whose availability is **not** subject to a feature flag will be used as the `X`-value in the typical `X.Y.Z` component of a semver@2.0.0 version string.
+4. The release engineer will generate relevant, point-in-time artifacts from this tag, such as the change log and data dictionary, among others.
+5. This tag will serve as the basis of a [GitHub Release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) to which the point-in-time artifacts are attached.
+6. The tag will also be used to identify the final _deployable_ artifacts, such as machine images, as part of the existing and future build, delivery, and deployment processes.
+
+### Proposed Solution: Unresolved Questions
+[Proposed Solution: Unresolved Questions]: #proposed-solution-unresolved-questions
+
+N/A.
+
+### Proposed Solution: Drawbacks
+[Proposed Solution: Drawbacks]: #proposed-solution-drawbacks
+
+None.
```

Comment:
We are at least in the short term adding a (admittedly small) additional step to our release process, which could be seen as a drawback.